examples/index.html

```diff
@@ -39,6 +39,9 @@ <h3 id=html>Custom Elements</h3>
     <a title=&lt;input-output&gt; href=/examples/input-output.html>&lt;input-output&gt;</a>
 
   <li>
+    <a title=&lt;multiple-choice&gt; href=/examples/multiple-choice>&lt;multiple-choice&gt;</a>
+
+  <li>
     <a title=&lt;x-flash&gt; href=/examples/x-flashy.html>&lt;x-flashy&gt;</a>
```

Comment:
i cringe a little when i see html entities still... could those be real characters if the attrs were wrapped with quotes? imho that'd be much nicer